photo/queries.py

```diff
@@ -24,7 +25,9 @@
 @strawberry.type
 class Query:
     @strawberry.field
-    def users(self, email: str = None) -> List[UserType]:
+    def users(self, user: uuid.UUID = None, email: str = None) -> List[UserType]:
+        if user:
```

Comment:
This should only use the user id, and must be restricted to the user, meaning that a user cannot see the details of another person

Reply:
Should this restriction to the user only being able to see his detais be implemented when implementing authentication?